skins/oasis/modules/OasisController.class.php

```diff
@@ -77,23 +76,18 @@ public static function onMakeGlobalVariablesScript(Array &$vars) {
 	public static function JsAtBottom(){
 		global $wgTitle;
 
-		// decide where JS should be placed (only add JS at the top for non-search Special and edit pages)
-		if (WikiaPageType::isSearch() || WikiaPageType::isForum()) {
-			// Remove this whole condition when AdDriver2.js is fully implemented and deployed
-
-			$jsAtBottom = true;
-		}
-		elseif ($wgTitle->getNamespace() == NS_SPECIAL || BodyController::isEditPage()) {
-			$jsAtBottom = false;
-		}
-		else {
-			$jsAtBottom = true;
-		}
-		return $jsAtBottom;
+		// decide where JS should be placed (only add JS at the top for legacy datatable-based special pages)
+		return !(
+			$wgTitle->isSpecial( 'MultiLookup' ) ||
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1067)